config/config/sc-config.yaml

```diff
@@ -27,6 +27,37 @@ objectStorage:
     opensearch: ${CK8S_ENVIRONMENT_NAME}-opensearch
     scFluentd: ${CK8S_ENVIRONMENT_NAME}-sc-logs
 
+  ## Restore object storage from off-site backups with rclone
+  ## Decrypt, destinations, and sources are automatically configured based on main and sync object storage configuration.
+  ## Note: The cronjobs setup by this must be manually run.
```

Comment:
Maybe we can link to the docs/restore/rclone.md here :)?

Reply:
Yes (it will point to the correct one once merged) :+1: